man/man8/zfs.8

```diff
@@ -268,6 +268,21 @@ zfs \- configures ZFS file systems
 .LP
 .nf
 \fBzfs\fR \fBdiff\fR [\fB-FHt\fR] \fIsnapshot\fR \fIsnapshot|filesystem\fR
+.fi
+
+.LP
+.nf
+\fB\fBzfs key -l\fR \fIfilesystem\fR | \fIvolume\fR\fR
```

Comment:
ZoL actually doesn't have any longopts handling at all. I can add it though, if it will help.

This (the wording of this command) is actually the biggest complaint I've received about the patch so far. I've heard from several people that they don't like that `key` isn't a verb like `create` or `destroy`. That said, I have been unable to come up with any alternatives that dont seem tedious (like `zfs loadkey`, zfs unloadkey`, `zfs changekey`. If you have any thoughts here now would probably be a good time while I'm cleaning up all of this other stuff.